fas2ipa/cli.py

```diff
@@ -248,6 +286,17 @@ def migrate_users(config, users, instances):
             groups_to_member_usernames[groupname].append(person["username"])
             if membership["role_type"] in ["administrator", "sponsor"]:
                 groups_to_sponsor_usernames[groupname].append(person["username"])
+                group_names = [g["name"] for g in person["memberships"]]
```

Comment:
This should not be in the `if` block, because it will be set only for sponsors and administrators, which gives a "local variable 'group_names' referenced before assignment" error on users who are just members.